staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -2181,6 +2187,19 @@ func describeJob(job *batchv1.Job, events *corev1.EventList) (string, error) {
 	})
 }
 
+func capIndexesList(indexes string, softLimit int) string {
+	ix := softLimit
+	for ; ix < len(indexes); ix++ {
+		if indexes[ix] == ',' {
```

Comment:
this will panic if len(indexes) < softLimit

Reply:
line 2192 doesn't let that happen